.github/workflows/push-trigger.yml

```diff
@@ -38,7 +38,7 @@ jobs:
             BUILD_ARTIFACT: 'pmp-ui'
       fail-fast: false
     name: ${{ matrix.SERVICE_NAME }}
-    uses: mosip/kattu/.github/workflows/docker-build.yml@master
+    uses: abhishek8shankar/kattu/.github/workflows/docker-build.yml@develop
```

Comment:
any specific reason to point to your repo and develop branch 

if no, please revert to mosip and master branch